### PR TITLE
Add mean aggregation for numerical features

### PR DESCRIPTION
**Pull request by jyx-su**

### CODE REVIEW

File: healthrex_ml/featurizers/starr_featurizers.py [hunk 1]
```diff
@@ -488,3 +488,271 @@ def _build_vocab(self, data):
...
+        SELECT 
+            observation_id, index_time, feature_type, feature, CAST(AVG(value) AS INT64) value 
+        FROM 
+            {self.feature_table_id}
+        WHERE 
+            feature_type IS NOT NULL
+        AND
+            feature IS NOT NULL
+        AND
+            feature LIKE 'NUM__%'
```
Comment (conorkcorbin) [inline healthrex_ml/featurizers/starr_featurizers.py:441]: This is awesome.  How do you handle missing values (ie when no numerical lab result value exists for an example in your dataset)?

**Review by conorkcorbin — COMMENTED** [review]
Hi Yixing this is great!  Since so much of this class is the same as `BagOfWordsFeaturizer`, wonder if instead of creating a new class definition altogether we add a flag somewhere as an attribute that if true executes additional logic all contained within `BagOfWordsFeaturizer`.  What do you think?

File: healthrex_ml/featurizers/starr_featurizers.py (hunk 1 above)
Comment (jyx-su) [reply healthrex_ml/featurizers/starr_featurizers.py:441]: That's a good point. I didn't handle the missing value cases as I excluded those cases without previous measurements during cohort building. As we're using sparse matric representation, I guess it defaults to zero?

Comment (conorkcorbin) [reply healthrex_ml/featurizers/starr_featurizers.py:441]: Yeah you're right, should just go to zero as written. Fine for now, maybe worth implementing some imputation at some other point down the road.

**Review by conorkcorbin — APPROVED** [review]
Looks good, thanks for updating!

**Pull request merged**